tests/CMakeLists.txt

```diff
@@ -25,6 +25,15 @@ if(GTEST_FOUND AND NOT MSVC)
   enable_testing()
 
   file(GLOB_RECURSE UNIT_TEST_SOURCE "cpp/*.cc" "cpp/*.h")
+  if(NOT USE_NGRAPH)
+    FOREACH(item ${UNIT_TEST_SOURCE})
+      IF(${item} MATCHES "ngraph")
+        LIST(REMOVE_ITEM UNIT_TEST_SOURCE ${item})
+      ENDIF(${item} MATCHES "ngraph")
+    ENDFOREACH(item)
+  else()
+    include_directories(${CMAKE_SOURCE_DIR}/src/ngraph)
+  endif()
```

Comment:
I know what you're doing this, and I agree, but it makes me want to throw up a little.  :(